datadog/import_datadog_integration_pagerduty_test.go

```diff
@@ -0,0 +1,46 @@
+package datadog
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestDatadogIntegrationPagerduty_import(t *testing.T) {
+	resourceName := "datadog_integration_pagerduty.pd"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckDatadogIntegrationPagerdutyDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccCheckDatadogIntegrationPagerdutyConfigImported,
+			},
+			resource.TestStep{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+const testAccCheckDatadogIntegrationPagerdutyConfigImported = `
+resource "datadog_integration_pagerduty" "pd" {
+  services 
```

Comment:
I'm not sure if I should name this service or services

Reply:
services matches the API and you are supporting a list of them. I'd stay with services